### PR TITLE
Add docs for `codespaces.openFiles`

### PR DESCRIPTION
**Pull request by joshaber**
Document `customizations.codespaces.openFiles`.

cc @craiglpeters

### CODE REVIEW

File: supporting.md [hunk 1]
```diff
@@ -97,19 +97,34 @@ If your Codespaces project needs additional permissions for other repositories,
...
 ```json
 "customizations": {
-		// Configure properties specific to Codespaces.
-		"codespaces": {
-			"repositories": {
-				"my_org/my_repo": {
-					"permissions": {
-						"issues": "write"
-					}
+	// Configure properties specific to Codespaces.
```
Comment (joshaber) [inline supporting.md:100]: This was two levels indented, I assume just an oversight.

Comment (samruddhikhandale) [reply supporting.md:100]: Thanks!

**Review by samruddhikhandale — APPROVED** [review]
Nice! 🚀

**Comment by samruddhikhandale** [comment]
@joshaber Could you help make the same change to https://github.com/devcontainers/spec/blob/main/docs/specs/supporting-tools.md ? Else, next time this change might get overridden.

**Comment by joshaber** [comment]
> @joshaber Could you help make the same change to https://github.com/devcontainers/spec/blob/main/docs/specs/supporting-tools.md ? Else, next time this change might get overridden.

Ah, yup! Updated in https://github.com/devcontainers/spec/pull/172.

**Pull request merged**